### PR TITLE
Add PnP Extraction Result to Planning Sheet Card

### PR DESCRIPTION
**Pull request by atGit2021**
[Monday](https://seed-company-squad.monday.com/boards/3451697530/views/78801638/pulses/7665475559?userId=37419707)

related: https://github.com/SeedCompany/cord-api-v3/pull/3334

## Release Notes

- **New Features**
  - Introduced a `Preview` component for file previews, enhancing user interaction.
  - Added `pnpExtractionResult` field to provide detailed extraction data in the engagement planning spreadsheet.

- **Bug Fixes**
  - Removed the `Preview` component from the `ProgressReportCard`, streamlining the user interface.

- **Enhancements**
  - Updated `PlanningSpreadsheet` to improve the visual presentation and interaction with file cards.

### CODE REVIEW

**Comment by atGit2021** [comment]
> @CarsonF , I tried putting the API PR in the comments above, but it doesn't look like that is fixing the issue of this PR not recognizing the schema change. I can't remember if this is the issue that requires a specific label to be selected above, but I don't see one that looks familiar. Everything works on my local, so I'm guessing I'm not referencing something correctly or I'm missing a label that will ignore the schema change.

Nevermind, got it working :)

File: src/components/Preview.tsx
Comment (rdonigian) [inline src/components/Preview.tsx:1]: I'd suggest we call this `FilePreview` and move it into a folder within components.

File: src/scenes/Engagement/LanguageEngagement/PlanningSpreadsheet/PlanningSpreadsheet.tsx
Comment (rdonigian) [inline src/scenes/Engagement/LanguageEngagement/PlanningSpreadsheet/PlanningSpreadsheet.tsx:1]: The header makes this component seemly overly verbose.   How about creating a new header component?

File: src/components/Preview.tsx
Comment (atGit2021) [reply src/components/Preview.tsx:1]: aborted this per our discoveries and discussions

**Review by CarsonF — APPROVED** [review]

**Pull request merged**